readme.txt

```diff
@@ -2,14 +2,17 @@
 Contributors: Godaddy, StarfieldTech, kurtpayne, asink
 Tags: debug, debugging, developer, development, performance, plugin, profiler, speed
 Requires at least: 3.3
-Tested up to: 4.7.2
+Tested up to: 4.6
```

Comment:
@EvanHerman Is there a known version compatibility issue with 4.7?

Reply:
No, we want to keep this 3 major releases back so the compat. notice on .org stays in tact - unless I have "major" release term mixed up...

Reply:
Oh right! That's a smart idea.